### PR TITLE
Server db

### PR DESCRIPTION
**Pull request by aku2nnya**
@Jvenci I have made some basic client/server/database setup and created the seed generator. If you can please review it, that would be great!
https://trello.com/c/FYjMJUtC/43-3-create-realistic-mock-data

### CODE REVIEW

File: client/src/components/carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    this.state = {
+
+    };
+  }
+
+  componentDidMount() {
+    this.getRequest();
+  }
+
+  getRequest() {
```
Comment (Jvenci) [inline client/src/components/carousel.jsx:17]: Will this getRequest method ever be used outside of componentDidMount()? If not, maybe move it inside the CDM.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 node_modules
-app.js
\ No newline at end of file
+carousel.app.js
```
Comment (Jvenci) [inline .gitignore:2]: Add your eslintrc.js file to your gitignore. It is a config file that doesn't need to be pushed to your repo

**Review by Jvenci — APPROVED** [review]
Looks good Toshi! I left some minor feedback on two items in the PR

File: client/src/components/carousel.jsx (hunk 1 above)
Comment (aku2nnya) [reply client/src/components/carousel.jsx:17]: Not 100% sure about what the best practices is here, but I would prefer to keep this separated as I believe it is easier to read.

File: .gitignore (hunk 2 above)
Comment (aku2nnya) [reply .gitignore:2]: Made the changes, thanks.

**Pull request merged**